apps/src/wallet/build.sh

```diff
@@ -1,5 +1,7 @@
 cargo +nightly build --target wasm32-unknown-unknown --release
 cp target/wasm32-unknown-unknown/release/*.wasm ../../../apps/wasm
+
+# compress the wasm artifact
 pushd ../../wasm
 wasm2wat wallet.wasm > wallet.wast
```

Comment:
@moshababo you need to run also
`wat2wasm` back again to get the reduced size `wasm`

Reply:
This is already happening, I haven't change that. 
You might need to expand github's code viewer.